volume_mount_builder.go

```diff
@@ -40,6 +39,37 @@ func (b VolumeMountBuilder) Build(resources []VolumeMount, inputs, outputs []str
 		pairsMap[parts[0]] = expandedPath
 	}
 
+	return pairsMap, nil
+}
+
+func (p PairsMap) Resolve(resourceName string) (string, error) {
+	if resourceLocation, ok := p[resourceName]; ok {
+		return resourceLocation, nil
+	}
+
+	resourceLocation, err := filepath.Abs(resourceName)
+	if err != nil {
+		return "", err
+	}
+	if _, err := os.Stat(resourceLocation); os.IsNotExist(err) {
+		return "", err
+	}
+
+	return resourceLocation, nil
+}
+
+const VolumeMountPoint = "/tmp/build"
+
+type VolumeMountBuilder struct{}
+
+
+func (b VolumeMountBuilder) Build(resources []VolumeMount, inputs, outputs []string) ([]DockerVolumeMount, error) {
```

Comment:
I think `inputs` and `outputs` are only passed in to construct the `PairsMap` struct. Maybe it make sense to create this outside of the `Build()` method and pass it in? Might make it easier to test.